install.sh

```diff
@@ -1,15 +1,97 @@
 #!/usr/bin/env sh
 
 sudo apt-get update
-sudo apt-get install -y ccache cmake clang-format python3-pip python3-setuptools libgmp3-dev parallel
-
+sudo apt-get install -y ccache cmake clang clang-format-7 python3-pip python3-setuptools libgmp3-dev g++ parallel
 python3 -m pip install --upgrade wheel
-python3 -m pip install --upgrade conan
 python3 -m pip install --upgrade trio
 
-conan profile new default --detect
-conan profile update settings.compiler.libcxx=libstdc++11 default
+if [ "$1" = "USE_CONAN" ]; then
+  echo "Using Conan build..."
+  echo "Installing Conan..."
+  python3 -m pip install --upgrade conan
+  conan profile new default --detect
+  conan profile update settings.compiler.libcxx=libstdc++11 default
+  . $HOME/.profile
+  return
+fi
+
+echo "Not using Conan..."
+echo "Installing dependecies..."
+
+sudo apt-get install -y \
+  libssl-dev \
+  autoconf \
+  automake \
+  libsnappy-dev \
+  zlib1g-dev \
+  libbz2-dev \
+  liblz4-dev \
+  libzstd-dev 
+
+echo "Installing Relic"
+cd /tmp
+git clone https://github.com/relic-toolkit/relic
+cd relic
+git checkout b984e901ba78c83ea4093ea96addd13628c8c2d0
+mkdir -p build
+cd build
+CC=/usr/bin/gcc CXX=/usr/bin/g++ cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DALLOC=AUTO -DWSIZE=64 -DRAND=UDEV -DSHLIB=ON -DSTLIB=ON -DSTBIN=OFF -DTIMER=HREAL -DCHECK=on -DVERBS=on -DARITH=x64-asm-254 -DFP_PRIME=254 -DFP_METHD="INTEG;INTEG;INTEG;MONTY;LOWER;SLIDE" -DCOMP="-O3 -funroll-loops -fomit-frame-pointer -finline-small-functions -march=native -mtune=native" -DFP_PMERS=off -DFP_QNRES=on -DFPX_METHD="INTEG;INTEG;LAZYR" -DPP_METHD="LAZYR;OATEP" ..
+CC=/usr/bin/gcc CXX=/usr/bin/g++ make
+sudo make install
+
+echo "Installing Cryptopp"
+cd /tmp
+git clone https://github.com/weidai11/cryptopp.git
+cd cryptopp
+git checkout CRYPTOPP_8_2_0
+make
+sudo make install
+
+echo "Installing rocksdb"
```

Comment:
This is a minor comment, don't feel like you need to change anything for it in this PR. 

The conditional around installing rocksdb and log4cplus in the travis preinstall script existed to speed up builds not using them. Now I think we should always use log4cplus, but building rocksdb does take a while, so we can speed up our travis jobs by not building it when not configured. Maybe it doesn't matter due to caching. Not sure... Either way, we may want to keep an eye on build times to see if they increase drastically without the conditional.

Reply:
FIxed